test/Microsoft.AspNet.Authentication.Test/OAuthBearer/OAuthBearerMiddlewareTests.cs

```diff
@@ -27,6 +26,8 @@ public class OAuthBearerMiddlewareTests
         {
             var server = CreateServer(options =>
             {
+                options.AutomaticAuthentication = true;
```

Comment:
Sure, that was just a general consideration regarding the default values for cookies (automatic) and bearer (manual), that I suggest to align if there's no strong reason to keep bearer manual by default.